src/status_im2/contexts/communities/menus/request_to_join/view.cljs

```diff
@@ -76,57 +77,64 @@
   [{:keys [permissions name id images
            can-join? can-request-access?
            requested-to-join-at]}]
-  (let [agreed-to-rules? (reagent/atom false)
-        is-open?         (not= 3 (:access permissions))]
-    (fn []
-      [rn/scroll-view {:style style/page-container}
-       [rn/view
-        {:style style/title-container}
+  (let [agreed-to-rules? (reagent/atom false)]
+    [:f>
+     (fn []
+       (let [{window-height :height} (rn/use-window-dimensions)
+             safe-area               (safe-area/use-safe-area)
```

Comment:
if its a drawer component and it's opened in bottom sheet i would say that bottom sheet should take care about safe-area

Reply:
Yes I agree, this is implemented before the new bottom sheet was ready so it's using the old bottom sheet. @alwx, I think the better solution is to implement this with the new bottom sheet and then we only fix this once.

Reply:
@J-Son89 it uses `status-im2.common.bottom-sheet.view` if that's what you're talking about. Is it the new one?

Reply:
@flexsurfer how do you propose to do this? The drawer component indeed cares about the safe-area and adds a spacing in the bottom of the screen but the problem is we actually don't need this spacing because this component contains a scroll-view that should all be scrollable (and unnecessary spacing makes it look weird)
In fact, I just implemented it similarly to `status-im2.contexts.chat.group-details.view` which is also a bottom sheet view.

Reply:
@ibrkhalil can you point us where the new one is :D

Reply:
@J-Son89 I do feel like `status-im2.common.bottom-sheet.view` is in fact the new one (the latest changes to it were done a week ago)

Reply:
yep it looks like it! Sorry for delayed response. It was created by @chrispader and seems he is first on the commits for it 👍

Reply:
Left it as it was because we need to control `safe-area` within the component for reasons explained above.